### ISSUE TITLE
`WPS226` false positive on fstring parts

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3548]
This code raises a violation:

```python
x = f'single {1}'
y = f'single {1}'
```

When `--max-string-usage=1` is provided.

Output:

```
  1:5      WPS226 Found string literal over-use: single  > 1
  x = f'single {1}'
      ^
```

This does not seem right, because strings that are part of `f` strings should not be counted by `WPS226`

PR would be rather easy :)

**Comment by F0RRZZ** [comment]
I'd like to take this
